app/contacts/new/page.js

```diff
@@ -0,0 +1,116 @@
+'use client';
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+
+export default function NewContact() {
+  const router = useRouter();
+
+  const [formData, setFormData] = useState({
+    name: '',
+    image_url: '',
+    email: '',
+    phone_number: ''
+  });
+
+  const [error, setError] = useState('');
+
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value
+    });
+  };
+
+  // Handles submit button
+  const handleSubmit = e => {
```

Comment:
don't be cheap with the param declaration
```suggestion
  const handleSubmit = event => {
```